qa/tasks/cephfs/kernel_mount.py

```diff
@@ -67,7 +68,10 @@ def mount(self, mntopts=[], createfs=True, check_status=True, **kwargs):
             if 'permission denied' in stderr.getvalue().lower():
                 pass
 
-
+        kmount_count = self.ctx.get('kmount_count', 0)
+        if kmount_count == 0:
+            self.enable_dynamic_debug()
+        self.ctx['kmount_count'] = kmount_count + 1
```

Comment:
I'm not sure it's a good idea to do this unconditionally. Debug logging can change the timing of things in subtle ways, which can mask race conditions and such. Maybe this should be something we can turn on when we're trying to troubleshoot a problem.

Reply:
Yeah, this sounds reasonable. I will add one option to do that.